test/testcafe/screenshot-test.js

```diff
@@ -0,0 +1,11 @@
+import { expect } from 'chai';
+import { statSync } from 'fs';
+
+fixture `Screenshot`
+    .page('https://google.com');
+
+test('Take screenshot', async t => {
+    var screenshotPath = await t.takeScreenshot();
```

Comment:
This topic is due to be sent to Armina today. So, if we could sort everything out before tomorrow, it would be cool.